optimum/exporters/onnx/model_configs.py

```diff
@@ -1035,8 +1035,7 @@ def inputs(self) -> Dict[str, Dict[int, str]]:
 
             if self.use_past_in_inputs:
                 self.add_past_key_values(common_inputs, direction="inputs")
-
         if self._behavior is ConfigBehavior.DECODER:
             common_inputs["encoder_outputs"] = {0: "batch_size", 1: "encoder_sequence_length"}
-
+            
```

Comment:
The code quality test didn't pass. Can you run the command `make style` in the root of the Optimum repo (you'll need to have the right packages installed with `pip install -e .[dev,exporters]`)?